### DISCUSSION TITLE
why is my nvim-tree.log 429GB big

### DISCUSSION THREAD

**Discussion by colorsu** [nvim-tree/nvim-tree.lua#2318] [Q&A] [answered]
![image](https://github.com/nvim-tree/nvim-tree.lua/assets/33319375/e5c2f8db-a571-474b-a5ff-0fe7c2a9436c)

As you can see that the picture I took from one of my compile server. The nvim-tree.log is huge.

**Comment by gegoune** [comment, marked as answer]
Check your logging settings `:h nvim-tree.log`.

**Reply by colorsu** [reply]
![image](https://github.com/nvim-tree/nvim-tree.lua/assets/33319375/49991357-47b2-402f-a09e-0f643f5c765a)
Thanks for your help? how can I get these configuration?

**Reply by colorsu** [reply]
![image](https://github.com/nvim-tree/nvim-tree.lua/assets/33319375/ea9933b8-3223-4223-b725-3db1088ef544)
Thank you~ I found the config which may cause this problem. I will close the logging function.
